taxi/urls.py

```diff
@@ -1,26 +1,58 @@
-from django.urls import path
+from argparse import Namespace
+from django.urls import path, include
+from django.contrib.auth import views as auth_views
 
-from .views import (
+from taxi.views import (
     index,
     CarListView,
     CarDetailView,
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    user_logout
```

Comment:
This file shouldn't me modified at all.
You already use build-in authentification methods by including 
`path("accounts/", include("django.contrib.auth.urls"))`